CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 #### 0.3.2
 
+- Deprecate setting the inverse active record association on cache hits. Set IdentityCache.never_set_inverse_association to true to avoid this.
```

Comment:
Thanks for the reminder.  I didn't add it while writing it because I didn't have a PR to get the number from.